scripts/import/remove_cosmic.pl

```diff
@@ -132,7 +127,7 @@ =head1 CONTACT
     AND     v.source_id = $source_id
```

Comment:
Are they ever in any set other than the cosmic set? This is probably a slow one, but deleting by set would be quick.

Reply:
Yes, I made quick simple changes. On e96 it took 1h with the old code